cmd/cainjector/main.go

```diff
@@ -35,8 +36,8 @@ func main() {
 	stopCh, exit := util.SetupExitHandler(util.GracefulShutdown)
 	defer exit() // This function might call os.Exit, so defer last
 
-	logf.InitLogs(flag.CommandLine)
```

Comment:
I think we don't want to stop using our `logf` wrapper here.

Reply:
@inteon  I think this should be okay. Our logger can then pull the initialized klog logger from context.